model_analyzer/cli/cli.py

```diff
@@ -128,12 +139,16 @@ def parse(self):
         args : Namespace
             Object that contains the parse CLI commands
             Used for the global options
+        input_args: List
```

Comment:
Can you clean up this docstring? This is under 'returns', which isn't true.

Reply:
Fixed